### ISSUE TITLE
Unable to proceed massaction "Update attributes" with required multiple select attribute 

### ISSUE THREAD

**Issue by tcurilla** [magento/magento2#11329]
### Preconditions

1. Magento ver. 2.2.0
2. PHP 7.0.24

### Steps to reproduce

1. clean install
2. create product attribute Multiple Select with id "test_multiselect" and set required values to true
3. fill some options values
4. create product with previously created multiple select attribute
5. save product
6. In  Adminhtml -> Catalog -> Products grid select product and do massaction "Update attributes"
7. click on SAVE

### Expected result

1. Saving must be processed without error

### Actual result

1. Error "The value of attribute "test_multiselect" must be set" is shown
![image](https://user-images.githubusercontent.com/1246371/31379465-a003fde6-adae-11e7-9d22-3634a85e279b.png)

### Possible solution 
In class method _\Magento\Framework\Data\Form\Element\Multiselect::getElementHtml()_
add "id" attribute to hidden element
```php
if ($this->getCanBeEmpty()) {
    $html .= '<input type="hidden" name="' . parent::getName() . '" value="" />';
}
```
change to 
```php
if ($this->getCanBeEmpty()) {
    $html .= '<input type="hidden" id="' . $this->getHtmlId() . '_hidden" name="' . parent::getName() . '" value="" />';
}
```

see _Magento_Catalog view/adminhtml/web/catalog/product.js_ _disableFieldEditMode/enableFieldEditMode_ methods

**Comment by magento-engcom-team** [comment]
@tcurilla, thank you for your report.
We've created internal ticket(s) MAGETWO-81398 to track progress on the issue.

**Comment by magento-engcom-team** [comment]
Hi @tcurilla the issue has been fixed in 2.2-develop branch and will be available with release 2.2.1 soon.

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-81467

**Comment by okorshenko** [comment]
Hi @tcurilla. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1323 by @magento-engcom-team in 2.3-develop branch
Related commit(s):
 - 809ed03393aeb55d51897c88561d9cdf1d0907a3
 - de9d15a247e165baef6f8138713077d80ac798ce
 - b322c098e8d39bfb2ef8a4287df7a5cfba4b2a84
 - 703d9e0a5d3367b6719b4445957438e066f497ed
 - ecfe9010ba5a0fc67e9c8a1fccbf202695d4ec3d
 - a90dbca28e49bf87b219bb9bcb091fb5a964e265
 - b4a7213a7efcd532d58bb600ad483709cc1e970d
 - 660c62a82a041c3f203c9e395e2bc62e7788940f
 - 8d48fd346467aa2267a97fc838d42c33a7399138
 - d0b525b16ba86b3bd59bb8e23adf3d9687de16df
 - 0dd84cd92c5c2d66471e3a4cef7768da33afdaf6
 - 6746da4c6b1b535043b79f7ccd5af91488008d34
 - 413f9394c659ca24dd942a23a908c8500088436e
 - 2eca1d0038d9ecf33c33f4bd9cff28ff4e20ceb3
 - f9976b9365b8d445f932274eb2160127918b888c
 - 4c77e4f5ebb0f7d82efb24a0f02ab7ef7773d8a5
 - 6c62727ca10389745b92ac683e5bb8deee7496f3
 - d8915032a9b00b93601f5dd8316272a5949b94f2
 - d10f55e4016b91dacb2991b44b5a65fbd5ccd7ab
 - e2ff943b8011aa6604e7280e9581f074ebd5ecb3
 - e27445da1e3039d898c53a3f8a9052094a892038
 - f0711edabca0ecc5daf384a838576822a6f4d4af
 - 79de0d6ad14e04ebcc645f6e336082cde6035423
 - a30789d9ec4bc536423b350509790cfc018c5691

The fix will be available with the upcoming 2.3.0 release.
